docs/usage/sat_bootprep.md

```diff
@@ -214,6 +214,56 @@ images:
   - Compute
 ```
 
+Here is an example of three IMS images built from the `secure-kubernetes-0.4.38-x86_64.squashfs:`
+and `secure-storage-ceph-0.4.38-x86_64.squashfs:` images provided by the `csm`
```

Comment:
It actually might be best to just say:

> Here is an example of three IMS images built from the Kubernetes image and the Ceph storage image provided by the `csm` product.

I just say that because the names you've shown here are the full names, which include a version. That version changes as the CSM version changes. I'm not even sure what the final image versions are going to be in Alice.

Reply:
Good point, I'll revise in the next commit.